app/views/games/play.html.slim

```diff
@@ -11,3 +11,10 @@ br
 = link_to 'Undo Round', undo_round_game_path(@game), class: 'btn btn-primary', method: 'delete'
 br
 = link_to 'Finish Game', finish_game_path(@game), class: 'btn btn-primary', method: 'patch'
+
+coffee:
+  ($ window).keypress (e) ->
+    if e.keyCode == 0 or e.keyCode == 32
+      e.preventDefault()
+      ($ "#next-button").click()
```

Comment:
`($ "#next-button")` is more idiomatic for coffeescript than `$('#next-button')`

Reason being: if we didn't call `click()` on the result, we would write `$ "#next-button"` rather than `$("#next-button")`. Since we do use the result, we wrap it in parens, which gives us `($ "#next-button").click()`